libs/perun/pipes/src/lib/consent-status-icon.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Consent, RichUser } from '@perun-web-apps/perun/openapi';
+
+@Pipe({ name: 'consentStatusIcon' })
+export class ConsentStatusIconPipe implements PipeTransform {
+  transform(user: RichUser, consents: Consent[]): string {
+    if (!consents) return;
+    const usersConsents = consents.filter((consent) => consent.userId === user.id);
+    // return latest consent (one with the highest id)
+    return usersConsents.reduce((a, b) => (a.id > b.id ? a : b)).status;
```

Comment:
This can throw an error if the array is empty.
![image](https://user-images.githubusercontent.com/23495969/196640777-bb30aa4e-f96d-4191-896a-61cf27a8ca1a.png)